android/app/build.gradle

```diff
@@ -110,6 +114,25 @@ android {
         versionName "1.3.46-1"
     }
 
+    flavorDimensions "default"
+    productFlavors {
+        production {
+        }
+        e2e {
+            // If  are building a version that won't be uploaded to the play store, we don't have to use production keys
+            // applies all non-production flavors
+            signingConfig signingConfigs.debug
+        }
```

Comment:
Can we apply the `.adhoc` suffix here too please? This way we can separate dev crashes from E2E in Firebase.